Gemfile.lock

```diff
@@ -1,26 +1,25 @@
 GEM
```

Comment:
a little issue here — `Gemfile.lock` has been committed, which it shouldn't have. (i.e. for now we want to stick to the previous version of the `Gemfile.lock` file.) To undo this, you can: 
- rollback to the previous version of the `Gemfile.lock`
- create a new commit with that new change (i.e. the new change being the switch from your “new” `Gemfile.lock` to our old `Gemfile.lock`)
- push the new change to your branch (this should update the PR accordingly)

I hope this helps. If it doesn't, or if it seems too cryptic, please ask your coaches or supervisor for help, or ping me and I can try to help! :)